src/app/shared/launch.service.ts

```diff
@@ -77,6 +77,36 @@ export abstract class LaunchService {
     return `nextflow run https://${workflowPath} -r ${versionName}`;
   }
 
+  /**
+   * This creates the planemo local init command
+   * @param path The GA4GH Tool's path
+   * @param versionName The ToolVersion's name
+   */
+  getPlanemoLocalInitString(workflowPath: string, versionName: string, primaryDescriptorPath: string, testParameterPath: string) {
+    return (
+      `wget -O foo.zip ${Dockstore.API_URI}${ga4ghPath}/tools/` +
+      encodeURIComponent('#workflow/' + workflowPath) +
+      `/versions/${versionName}/GALAXY/files?format=zip` +
```

Comment:
I think the versionName needs to be encoded as well; I noticed some versions with `/` in them when testing the metrics stuff.

Reply:
Oy :grimacing:  (more about the versions with a slash in them, than having to encode)